test/wasm2js/conversions-emscripten-modified.wast

```diff
@@ -0,0 +1,564 @@
+(module
```

Comment:
There should be output files to check in with this, something like `FILENAME.2asm.js.*`.

But, for a test, maybe add just a minimal file with one fptoint operation? I don't think we need to add coverage for the entire set of instructions, which the pass has tests for. We mainly want to test that the pass runs. Unless there is a benefit I'm missing?

Reply:
Ah yes, I forgot to check that output file in. but... ok why didn't the CI tests fail then? 🤔 
I'll switch to a minimal file.

Reply:
Hmm,

https://github.com/WebAssembly/binaryen/blob/cc97853b5f03e8c8441159e68ecb2262beee166f/scripts/test/wasm2js.py#L67-L68

It looks like we take the lack of an expected file to mean there is nothing there to test. That was needed for spec tests, as not all pass in wasm2js, but we shouldn't be doing that for normal tests... I'll investigate.